tachyon/base/parallelize.h

```diff
@@ -105,6 +105,53 @@ auto ParallelizeMap(Container& container, Callable callback,
                                    std::move(callback));
 }
 
+// Splits the |container| by |chunk_size| and maps each chunk using the provided
+// |callback| in parallel. Each callback's return value is collected into a
+// vector which is then returned.
+// See parallelize_unittest.cc for more details.
+template <typename Container, typename Callable,
+          typename FunctorTraits = internal::MakeFunctorTraits<Callable>,
+          typename RunType = typename FunctorTraits::RunType,
+          typename ReturnType = typename FunctorTraits::ReturnType,
+          typename ArgList = internal::ExtractArgs<RunType>,
+          typename SpanTy = internal::GetType<0, ArgList>,
+          typename T = typename SpanTy::value_type,
+          size_t ArgNum = internal::GetSize<ArgList>>
+std::pmr::vector<ReturnType> PmrParallelizeMapByChunkSize(Container& container,
```

Comment:
Please remove `PmrParallelizeMapByChunkSize()` and `PmrParallelizeMap()`. The size of the vector will be the number of threads, which typically is less than 64, so there is no merit in keeping these functions.

Reply:
`PmrParallelizeMap()` is used in [`DoEvaluate`](https://github.com/kroma-network/tachyon/blob/e8558ecddb6b36dfc0d63e78cf5e8c940844eb06/tachyon/math/polynomials/univariate/univariate_dense_coefficients.h#L267)